docs/orleans/host/client.md

```diff
@@ -25,7 +26,21 @@ Despite these detractors, co-hosting client code with grain code is a popular op
 
 If hosting using the [.NET Generic Host](../../core/extensions/generic-host.md), the client will be available in the host's [dependency injection](../../core/extensions/dependency-injection.md) container automatically and can be injected into services such as [ASP.NET controllers](/aspnet/core/mvc/controllers/actions) or <xref:Microsoft.Extensions.Hosting.IHostedService> implementations.
 
-Alternatively, a client interface such as <xref:Orleans.IGrainFactory> or <xref:Orleans.IClusterClient> can be obtained from either <xref:Microsoft.Extensions.Hosting.IHost> or <xref:Orleans.Hosting.ISiloHost>:
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-7-0"
+<!-- markdownlint-enable MD044 -->
+
+Alternatively, a client interface such as <xref:Orleans.IGrainFactory> can be obtained from <xref:Microsoft.Extensions.Hosting.IHost> (or the <xref:Orleans.IClusterClient.ServiceProvider?displayProperty=nameWithType>):
```

Comment:
`IClusterClient` *is* an `IGrainFactory`, so you can use an `IClusterClient` as an `IGrainFactory` immediately.